specificatie/openapi.yaml

```diff
@@ -19,6 +19,7 @@ tags:
 paths:
   /ingeschrevenpersonen/{burgerservicenummer}/verblijfplaatshistorie:
     get:
+      deprecated: true
```

Comment:
zoals besproken moet deze in eerste instantie niet deprecated worden. pas na de pilot kan gekozen worden welke variant (get of post) overblijft

Reply:
Aangepast